test/test_datasets.py

```diff
@@ -141,76 +141,86 @@ def test_places365_repr_smoke(self):
             self.assertIsInstance(repr(dataset), str)
 
 
-class STL10Tester(DatasetTestcase):
-    @contextlib.contextmanager
-    def mocked_root(self):
-        with stl10_root() as (root, data):
-            yield root, data
+class STL10Tester(datasets_utils.ImageDatasetTestCase):
```

Comment:
To keep it in sync with the other test cases:

```suggestion
class STL10TestCase(datasets_utils.ImageDatasetTestCase):
```